community/hsetroot/sources

```diff
@@ -0,0 +1 @@
+git+https://github.com/himdel/hsetroot/
```

Comment:
Why git?

Reply:
It's a git repo, what do you mean

Reply:
Why not a release tarball? https://github.com/himdel/hsetroot/releases

Reply:
👍 ok